test/gov/apireg.sh

```diff
@@ -173,47 +139,8 @@ fi
 if [ "$HA" == "1" ]; then
     if [ "$PATTERN" = "sns" ] || [ "$PATTERN" = "spws" ]; then
 
-        read -d '' pwsservice <<EOF
-{
-  "url": "https://bluehorizon.network/services/no-such-service",
-  "organization": "e2edev@somecomp.com",
-  "version": "1.0.0",
-  "attributes": [
-    {
-      "type": "ComputeAttributes",
```

Comment:
We need to keep this test so you need to change the attribute to something other than compute, like userinput.